lib/multicell_location/services/skyhook_integration.c

```diff
@@ -8,6 +8,7 @@
 #include <stdio.h>
 #include <modem/lte_lc.h>
 #include <modem/modem_info.h>
```

Comment:
Looks like this can also be removed now, along with any Kconfig dependency

Reply:
Removed.